### PR TITLE
Rename sym and (un)wrap to key and (un)seal for more clarity

### PR DESCRIPTION
**Pull request by yannham**

### CODE REVIEW

File: src/term.rs [hunk 1]
```diff
@@ -432,8 +434,8 @@ impl Term {
             Term::Enum(_) => Some("Enum"),
             Term::Record(..) | Term::RecRecord(..) => Some("Record"),
             Term::Array(..) => Some("Array"),
-            Term::Sym(_) => Some("Sym"),
-            Term::Wrapped(_, _) => Some("Wrapped"),
+            Term::SealingKey(_) => Some("Sym"),
+            Term::Sealed(_, _) => Some("Wrapped"),
```
[the remark this answers is not in the document]
Comment (yannham) [reply src/term.rs]: Yes, good catch.

File: src/pretty.rs [hunk 2]
```diff
@@ -671,12 +671,12 @@ where
...
                 .group(),
 
-            Sym(sym) => allocator
+            SealingKey(sym) => allocator
                 .text(format!("# <symbol: {}>", sym))
                 .append(allocator.hardline()),
 
             // TODO
-            Wrapped(_i, _rt) => allocator.text("# <wraped>").append(allocator.hardline()),
+            Sealed(_i, _rt) => allocator.text("# <wraped>").append(allocator.hardline()),
```
Comment (ebresafegaga) [inline src/pretty.rs:679]: Also, should this be `"# <sealed>"`?

File: src/term.rs [hunk 3]
```diff
@@ -484,8 +486,8 @@ impl Term {
             }
             Term::Record(..) | Term::RecRecord(..) => String::from("{ ... }"),
             Term::Array(..) => String::from("[ ... ]"),
-            Term::Sym(_) => String::from("<sym>"),
-            Term::Wrapped(_, _) => String::from("<wrapped>"),
+            Term::SealingKey(_) => String::from("<sym>"),
+            Term::Sealed(_, _) => String::from("<wrapped>"),
```
Comment (ebresafegaga) [inline src/term.rs:490]: Same thing here

**Comment by yannham** [comment]
Bad search and replace skills, it seems. Thanks.

**Review by ebresafegaga — APPROVED** [review]
Looks good now

**Pull request merged**